src/server/kiwix-serve.cpp

```diff
@@ -162,9 +162,7 @@ void introduceTaskbar(string& content, const string& humanReadableBookId)
     content = appendToFirstOccurence(
         content,
         "<head>",
-        replaceRegex(
-            RESOURCE::include_html_part, humanReadableBookId, "__CONTENT__")
-        + (noLibraryButtonFlag
+            RESOURCE::include_html_part + (noLibraryButtonFlag
```

Comment:
You're not replacing `___CONTENT__` anymore. Is it wanted ?

Reply:
No I remplace by __CONTENT_ESCAPE__ (line 184)
And include.html.part have also been update (no more __CONTENT__ only __CONTENT_ESCAPE__ )

Reply:
My bad, I was looking into `taskbar.html.part`.